src/desktop.cpp

```diff
@@ -27,15 +27,15 @@ dxp_desktop::dxp_desktop (
   this->pixmap_width = dexpo_width;
   this->pixmap_height = dexpo_height;
 
+  float screen_ratio = float (this->width) / this->height; ///< width/height
```

Comment:
In this case not really, as the variable scope is very small.

screen_ratio doesn't convey whether it is width/height or height/width. As the comment is in the doxygen format it can be parsed by Language Server and shown to programmer when he hovers over variable